### PR TITLE
Add function to calculate the result of picks

### LINKED ISSUE #28: Add 'analyze picks' function

### PR DESCRIPTION
**Pull request by JuKahl**
🚧  WORK IN PROGRESS 🚧

The function to calculate whether the picks stored in the database are right or wrong finally works. Currently only console.logs are generated as a result of the calculation -  this will be replaced by corresponding entries in the database.

### CODE REVIEW

**Review by DNaunin — APPROVED** [review]
Good progress! 🎉

File: components/matchBox/TeamNames.tsx [hunk 1]
```diff
@@ -6,9 +6,16 @@ export type Teams = {
   away: string;
   handlePick: any;
```
Comment (mewiemers) [inline components/matchBox/TeamNames.tsx:7]: did you hover on your handle pick to see your type?

File: logic/result.tsx [hunk 2]
```diff
@@ -0,0 +1,44 @@
+import { Match, addPick } from "../utils/api";
```
Comment (mewiemers) [inline logic/result.tsx]: addPick is not used, remember this before merging

File: logic/result.tsx [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+      .then((response) => response.json())
+      .then(setMatches);
+    fetch("/api/picks")
+      .then((response) => response.json())
+      .then(setPicks);
+  }, []);
+  if (!matches) {
+    return <div> Loading...</div>;
+  }
+  if (!picks) {
```
Comment (mewiemers) [inline logic/result.tsx]: is it possible to use one statement? because the return is the same

**Review by mewiemers — COMMENTED** [review]
great job! 🏈

File: components/matchBox/TeamNames.tsx (hunk 1 above)
Comment (JuKahl) [reply components/matchBox/TeamNames.tsx:7]: I just removed the type declaration and hovered over handlepick - it displays "any" as type.

**Review by fre-ben — APPROVED** [review]
LGTM ⚽

File: logic/result.tsx (hunk 3 above)
Comment (fre-ben) [reply logic/result.tsx]: That's a good idea, I think. Something like:
```suggestion
  if (!matches || !picks) {
    return <div> Loading...</div>;
  }
```

**Review by m-wiemers — APPROVED** [review]
gj 🚀

**Pull request merged**